schemas/chemotherapy.json

```diff
@@ -50,19 +50,31 @@
       }
     },
     {
-      "name": "chemotherapy_drug_name",
-      "description": "Name of agent or drug administered to patient as part of the chemotherapy treatment regimen.",
+      "name": "drug_rxnormcui",
+      "description": "The unique RxNormID assigned to the treatment regimen drug.",
       "valueType": "string",
-      "restrictions": {
-        "required": true,
-        "codeList": ["Placeholder list 1", "Need list", "Still need it"]
+      "meta": {
+        "validationDependency": true,
+        "core": true,
+        "notes": "This field uses a controlled vocabulary gathered from the RxNorm database of drug names provided by the NIH (https://www.nlm.nih.gov/research/umls/rxnorm).",
+        "displayName": "RxNormCUI"
       },
+      "restrictions": {
+        "required": true
+      }
+    },
+    {
+      "name": "drug_name",
```

Comment:
to confirm, submitters can update this field, it's not an id

Reply:
@blabadi it should be a valid rxcui/name combination though.  Since there are multiple of that (i.e 1 id can have many names), i think this is ok to update to a data submitter can indicate the most correct version of their drug_name.  @hknahal?

Reply:
@rosibaj Yes, I agree, the "drug_name" field can be updated by the data submitter, as long as the rxcui and drug name are a valid combination.